test/e2e/config/azure-dev.yaml

```diff
@@ -96,22 +96,22 @@ providers:
       targetName: "cluster-template-custom-vnet.yaml"  
 
 variables:
-  KUBERNETES_VERSION: "${KUBERNETES_VERSION:-v1.21.2}"
+  KUBERNETES_VERSION: "${KUBERNETES_VERSION:-v1.22.1}"
   # using a different version for windows because of an issue on azure cloud provider
   # that only affects windows and external load balancer  
   # https://github.com/kubernetes-sigs/cloud-provider-azure/issues/706
-  WINDOWS_KUBERNETES_VERSION: "${WINDOWS_KUBERNETES_VERSION:-v1.19.11}"
+  WINDOWS_KUBERNETES_VERSION: "${WINDOWS_KUBERNETES_VERSION:-v1.22.1}"
```

Comment:
we can remove this now, let's follow up on that after this merges though

Reply:
I'll do a followup PR after this is done